src/module_3/module3_statistical_learning.md

```diff
@@ -0,0 +1,1368 @@
+```python
+import os
+import numpy as np
+import pandas as pd
+from summarytools import dfSummary
```

Comment:
is this a python module? Never heard of it, but doesn't seem to use the best naming convention 😆